# Практическая работа №2: Обработка выборочных данных. Нахождение интервальных оценок параметров распределения. Проверка статистической гипотезы о нормальном распределении

Выполнила студентка гр. 1384 Шиняева Анастасия. Вариант №4

## Цель работы

Получение практических навыков вычисления интервальных статистических оценок параметров распределения выборочных данных и проверки «справедливости» статистических гипотез.

## Основные теоретические положения

**Доверительный интервал для математического ожидания при неизвестном стандартном отклонении:**

$ \bar{x} \pm t_{\gamma, N-1} \cdot \frac{S}{\sqrt{N}} $

где $ \bar{x} $  — выборочное среднее, $ S $  — исправленное стандартное отклонение, $ N $ — объём выборки, $ t_{\gamma, N-1} $ — квантиль распределения Стьюдента для уровня доверия $ \gamma $ и степеней свободы $ N-1 $.

**Квантиль распределения Стьюдента:**

$ t_{\gamma, N-1} = S_{N-1}^{-1}\left(\frac{1 + \gamma}{2}\right) $

где $ S_{N-1}^{-1} $ — обратная функция распределения Стьюдента с $ N-1 $ степенями свободы.

**Доверительный интервал для среднеквадратичного отклонения при неизвестном стандартном отклонении:**

Исходное неравенство:

$ S - \delta < \sigma < S + \delta $

Преобразование к виду:

$ S(1 - \delta / S) < \sigma < S(1 + \delta / S) $

Пусть $ q = \delta / S $, тогда неравенство примет вид:

$ S(1 - q) < \sigma < S(1 + q) $

Если $ q < 1 $, то неравенство для обратных величин:

$ \frac{1}{S(1 + q)} < \frac{1}{\sigma} < \frac{1}{S(1 - q)} $

**Критерий Пирсона:**

$ \chi^2_{набл} = \sum_{i = 1}^k\frac{n^2_i}{n'_i} - n$

**Статистической гипотезой** называют гипотезу о виде неизвестного распределения исследуемой случайной величины или о значении параметров известных распределений.

Выдвинутую гипотезу называют **нулевой или основной гипотезой** и обозначают ${H_0}$.

**Конкурирующей или альтернативной** называют гипотезу ${Н_1}$, которая противоречит основной гипотезе.

**Ошибка первого рода** состоит в том, что нулевая гипотеза отвергается, хотя она верна.

**Ошибка второго рода** состоит в том, что нулевая гипотеза принимается, хотя она неверна.

Вероятность совершения ошибки первого рода называют **уровнем значимости** и обозначают через $\alpha$.

Вероятность совершения ошибки второго рода обозначают $\beta$.

Величина $1 - \beta$ - вероятность того, что гипотеза отвергается, когда она неверна, называют **мощностью критерия**.

**Областью принятия** нулевой гипотезы называют совокупность значения критерия, при которых нулевую гипотезу принимают.

**Критической областью** называют совокупность значений критерия, при которых нулевую гипотезу отвергают.

## Постановка задачи

Для заданной надежности определить (на основании выборочных данных и результатов выполнения практической работы №2) границы доверительных интервалов для математического ожидания и среднеквадратичного отклонения случайной величины. Проверить гипотезу о нормальном распределении исследуемой случайной величины с помощью критерия Пирсона χ2. Дать содержательную интерпретацию полученным результатам.

## Выполнение работы

Перенесем необходимые данные из первых практических работ для выполнения текущей.

Сначала проведем вычисления для столбца E.

In [19]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

data = pd.read_csv('/data.csv', header=None, names=['nu', 'E']) # Выборка

ranked_data = data['E'].sort_values() # Ранжированный ряд

min_value = data['E'].min() # Минимальное значение
max_value = data['E'].max() # Максимальное значение

bin_edges = 7 # Количество интервалов
h = (max_value - min_value) / bin_edges # Ширина интервала

intervals = [min_value + i * h for i in range(bin_edges + 1)] # Границы интервала

absolute_freq, bin_edges = np.histogram(data['E'], bins=bin_edges) # Абсолютная частота
relative_freq = absolute_freq / len(data) # Относительная частота

bin_mid = (bin_edges[:-1] + bin_edges[1:]) / 2 # Середины интервалов

cumulative_absolute_freq = np.cumsum(absolute_freq) # Накопленная абсолютная частота
cumulative_relative_freq = np.cumsum(relative_freq) # Накопленная относительная частота

C = bin_mid[len(bin_mid) // 2]
u = (bin_mid - C) / h # Условные варианты

n_u1 = relative_freq * u**1 # Условные моменты
n_u2 = relative_freq * u**2
n_u3 = relative_freq * u**3
n_u4 = relative_freq * u**4

n_u1_4 = relative_freq * (u+1)**4 # Сумма условных моментов

M_1 = sum(n_u1)  # Условные эмпирические моменты
M_2 = sum(n_u2)
M_3 = sum(n_u3)
M_4 = sum(n_u4)

m_1 = 0 # Центральные эмпирические моменты
m_2 = (M_2 - M_1**2)*h**2
m_3 = (M_3 - 3*M_1*M_2 + 2*M_1**3)*h**3
m_4 = (M_4 - 4*M_1*M_3 + 6*M_1**2*M_2 - 3*M_1**4)*h**4

x_avg_b = M_1*h + C # Выборочное среднее
D_b = m_2 # Выборочная дисперсия

sigma_b = np.sqrt(D_b) # Выборочное СКО

s_2 = len(data['E']) / (len(data['E']) - 1) * D_b # Исправленная выборочная дисперсия
s = np.sqrt(s_2) # Исправленное СКО

A_s = m_3 / s**3 # Статистическая оценка коэффициента асимметрии
E = m_4 / s**4 - 3 # Статистическая оценка коэффициента эксцесса

Вычислим точность и доверительный интервал для математического ожидания при неизвестном среднеквадратичном отклонении при заданном объёме выборки для доверительной точности $ γ ∈ {0.95,0.99} $.

Объем выборки $N$ = количество интервалов $k$ = 7

In [20]:
from scipy.stats import t

gamma_values = [0.95, 0.99]
k = 7

for gamma in gamma_values:
    t_gamma = t.ppf((1 + gamma) / 2, df=k-1)

    margin_of_error = t_gamma * s / np.sqrt(k)
    confidence_interval = (x_avg_b - margin_of_error, x_avg_b + margin_of_error)

    print(f"Точность и доверительный интервал для математического ожидания для γ = {gamma}: {margin_of_error}, {confidence_interval}")

Точность и доверительный интервал для математического ожидания для γ = 0.95: 22.762545975369093, (104.40428570779923, 149.92937765853742)
Точность и доверительный интервал для математического ожидания для γ = 0.99: 34.48857413735256, (92.67825754581577, 161.65540582052088)


Выводы: для уровня доверия 0.95 интервал уже (45.525), что указывает на более точную оценку, но с меньшей уверенностью, для уровня доверия 0.99 интервал шире (68.977), что обеспечивает большую уверенность, но снижает точность оценки.

Для вычисления границ доверительного интервала для среднеквадратичного отклонения определим значение $q$ при заданных $γ$ и $n$ и построим доверительные интервалы.

Из таблицы в Приложении 7:

Для $γ$ = 0.95 $q$ = 0,92.

Для $γ$ = 0.99 $q$ = 1,62.

In [21]:
q_values = {0.95: 0.92, 0.99: 1.62}

for gamma, q in q_values.items():
    if gamma == 0.95:
        s_trust_interval = [s * (1 - q), s * (1 + q)]
    elif gamma == 0.99:
        s_trust_interval = [0, s * (1 + q)] # 0 чтобы не было отрицательных значений

    print(f"Доверительный интервал для среднеквадратичного отклонения для γ = {gamma}: ({s_trust_interval[0]}, {s_trust_interval[1]})")

Доверительный интервал для среднеквадратичного отклонения для γ = 0.95: (1.9689809693576168, 47.255543264582826)
Доверительный интервал для среднеквадратичного отклонения для γ = 0.99: (0, 64.48412674646198)


Выводы: для уровня доверия 0.95 интервал уже (45.286), но всё ещё достаточно широкий, что указывает на значительную неопределённость, для уровня доверия 0.99 интервал значительно шире (64.484), что обеспечивает большую уверенность, но снижает точность оценки.

Проверим гипотезу о нормальности заданного распределения с помощью критерия Пирсона. Для этого найдем теоретические частоты и вычислим наблюдаемое значение критерия.

In [22]:
from scipy.stats import norm

intervals = list(bin_edges)
norm_intervals = [(boundary - x_avg_b) / s for boundary in intervals]
norm_intervals[0] = -np.inf
norm_intervals[-1] = np.inf

theor_prob = [
    norm.cdf(norm_intervals[i]) - norm.cdf(norm_intervals[i - 1])
    for i in range(1, len(norm_intervals))
]
print("Теоретические вероятности:", theor_prob, sum(theor_prob))

theor_freq = [prob * len(data['E']) for prob in theor_prob]
print("\nТеоретические частоты:", theor_freq, sum(theor_freq))

squares_diff = [
    (observed - theor) ** 2
    for observed, theor in zip(absolute_freq, theor_freq)
]
print("\nКвадраты разностей между наблюдаемыми и теоретическими частотами", squares_diff)

relative_squares_diff = [
    squares_diff / theor
    for squares_diff, theor in zip(squares_diff, theor_freq)
]
print("\nОтносительные квадраты разностей", relative_squares_diff)

squares_observed_freq = [ni**2 for ni in absolute_freq]
print("\nКвадраты наблюдаемых частот", squares_observed_freq)

relative_squares_observed_freq = [
    ni**2 / ni_prime
    for ni, ni_prime in zip(absolute_freq, theor_freq)
]
print("\nОтносительные квадраты наблюдаемых частот:", relative_squares_observed_freq, sum(relative_squares_observed_freq))

observed_X2 = sum(relative_squares_diff)
print("\nНаблюдаемое значение критерия Пирсона:", observed_X2)

Теоретические вероятности: [0.03341514770218198, 0.09805593309480617, 0.21085805695664456, 0.27833847577096815, 0.22560661611403898, 0.11226082762176182, 0.04146494273959833] 1.0

Теоретические частоты: [3.37492991792038, 9.903649242575424, 21.2966637526211, 28.112186052867784, 22.786268227517937, 11.338343589797944, 4.187959216699431] 101.0

Квадраты разностей между наблюдаемыми и теоретическими частотами [2.640852771670263, 0.8165819536071381, 18.461319403088034, 8.339469393251347, 1.4731448155324511, 0.4377892051614716, 1.4112471005411258]

Относительные квадраты разностей [0.7824911437857492, 0.08245263272215683, 0.8668643885977632, 0.2966496229630858, 0.06465055185093456, 0.03861138989961348, 0.336977278793403]

Квадраты наблюдаемых частот [25, 81, 289, 961, 576, 144, 9]

Относительные квадраты наблюдаемых частот: [7.407561225865369, 8.178803390146733, 13.570200635976663, 34.1844635700953, 25.278382324333, 12.70026780010167, 2.149018062093972] 103.46869700861271

Наблюдаемое значе

Оформим вычисления в таблицу:

Таблица 1

|    $ i $   	|           $ [x_{i}, x_{i+1}) $           	| $ n_i $ 	|       $ p_i $       	|      $ n'_i $      	| $ (n_i - n'_i)^2 $ 	| $ \cfrac{(n_i - n'_i)^2}{n'_i} $ 	| $ n_i^2 $ 	| $ \cfrac{n_i^2}{n'_i} $ 	|
|:----------:	|:----------------------------------------:	|:-------:	|:-------------------:	|:------------------:	|:------------------:	|:--------------------------------:	|:---------:	|:-----------------------:	|
|      1     	|         [64.5, 82.05714285714285)        	|    5    	| 0.03341514770218198 	|  3.37492991792038  	|  2.640852771670263 	|        0.7824911437857492        	|     25    	|    7.407561225865369    	|
|      2     	|  [82.05714285714285, 99.61428571428571)  	|    9    	| 0.09805593309480617 	|  9.903649242575424 	| 0.8165819536071381 	|        0.08245263272215683       	|     81    	|    8.178803390146733    	|
|      3     	|  [99.61428571428571, 117.17142857142858) 	|    17   	| 0.21085805695664456 	|  21.2966637526211  	| 18.461319403088034 	|        0.8668643885977632        	|    289    	|    13.570200635976663   	|
|      4     	| [117.17142857142858, 134.72857142857143) 	|    31   	| 0.27833847577096815 	| 28.112186052867784 	|  8.339469393251347 	|        0.2966496229630858        	|    961    	|     34.1844635700953    	|
|      5     	| [134.72857142857143, 152.28571428571428) 	|    24   	| 0.22560661611403898 	| 22.786268227517937 	| 1.4731448155324511 	|        0.06465055185093456       	|    576    	|     25.278382324333     	|
|      6     	| [152.28571428571428, 169.84285714285716) 	|    12   	| 0.11226082762176182 	| 11.338343589797944 	| 0.4377892051614716 	|        0.03861138989961348       	|    144    	|    12.70026780010167    	|
|      7     	|        [169.84285714285716, 187.4]       	|    3    	| 0.04146494273959833 	|  4.187959216699431 	| 1.4112471005411258 	|         0.336977278793403        	|     9     	|    2.149018062093972    	|
| $ \Sigma $ 	|                     -                    	|   101   	|         1.0         	|        101.0       	|          -         	|        2.4686970086127062        	|     -     	|    103.46869700861271   	|

Докажем, что $ \chi^2_{набл} = \sum_{i = 1}^k\frac{n^2_i}{n'_i} - n$ и проконтролируем корректность вычисления $ \chi^2_{набл} $.

$ \chi^2_{набл} = $

$ = \sum_{i=1}^{k} \frac{(n_i - n'_i)^2}{n'_i} = \sum_{i=1}^{k} \frac{n_i^2 - 2 n'_i n_i + {n'_i}^2}{n'_i} = \sum_{i=1}^{k} (\frac{n_i^2}{n'_i} - 2 \frac{n'_i n_i}{n'_i} + \frac{{n'_i}^2}{n'_i}) = \sum_{i=1}^{k} (\frac{n_i^2}{n'_i} - 2 n_i + n'_i) = \sum_{i=1}^{k} \frac{n_i^2}{n'_i} - 2 \sum_{i=1}^{k} n_i + \sum_{i=1}^{k} n'_i = \sum_{i=1}^{k} \frac{n_i^2}{n'_i} - 2 n_i + n_i = $

$ = \sum_{i=1}^{k} \frac{n_i^2}{n'_i} - n $

In [23]:
print(sum(relative_squares_observed_freq) - sum(absolute_freq), "=", observed_X2)

2.4686970086127076 = 2.4686970086127062


По заданному уровню значимости $α=0.05$ и числу степеней свободы $df$ найдем критическую точку $ \chi^2_{крит} $ и сравним с наблюдаемым значением.

$ df = k - 3 = 4 $

Из таблицы в Приложении 5:

$ \chi^2_{крит} = 9.5 $

$ \chi^2_{набл} < \chi^2_{крит} (2.5 < 9.5) $

Выводы: гипотеза о нормальности распределения не отвергается.

Проведем аналогичные вычисления для столбца nu.

In [24]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

data = pd.read_csv('/data.csv', header=None, names=['nu', 'E']) # Выборка

ranked_data = data['nu'].sort_values() # Ранжированный ряд

min_value = data['nu'].min() # Минимальное значение
max_value = data['nu'].max() # Максимальное значение

bin_edges = 7 # Количество интервалов
h = (max_value - min_value) / bin_edges # Ширина интервала

intervals = [min_value + i * h for i in range(bin_edges + 1)] # Границы интервала

absolute_freq, bin_edges = np.histogram(data['nu'], bins=bin_edges) # Абсолютная частота
relative_freq = absolute_freq / len(data) # Относительная частота

bin_mid = (bin_edges[:-1] + bin_edges[1:]) / 2 # Середины интервалов

cumulative_absolute_freq = np.cumsum(absolute_freq) # Накопленная абсолютная частота
cumulative_relative_freq = np.cumsum(relative_freq) # Накопленная относительная частота

C = bin_mid[len(bin_mid) // 2]
u = (bin_mid - C) / h # Условные варианты

n_u1 = relative_freq * u**1 # Условные моменты
n_u2 = relative_freq * u**2
n_u3 = relative_freq * u**3
n_u4 = relative_freq * u**4

n_u1_4 = relative_freq * (u+1)**4 # Сумма условных моментов

M_1 = sum(n_u1)  # Условные эмпирические моменты
M_2 = sum(n_u2)
M_3 = sum(n_u3)
M_4 = sum(n_u4)

m_1 = 0 # Центральные эмпирические моменты
m_2 = (M_2 - M_1**2)*h**2
m_3 = (M_3 - 3*M_1*M_2 + 2*M_1**3)*h**3
m_4 = (M_4 - 4*M_1*M_3 + 6*M_1**2*M_2 - 3*M_1**4)*h**4

x_avg_b = M_1*h + C # Выборочное среднее
D_b = m_2 # Выборочная дисперсия

sigma_b = np.sqrt(D_b) # Выборочное СКО

s_2 = len(data['nu']) / (len(data['nu']) - 1) * D_b # Исправленная выборочная дисперсия
s = np.sqrt(s_2) # Исправленное СКО

A_s = m_3 / s**3 # Статистическая оценка коэффициента асимметрии
E = m_4 / s**4 - 3 # Статистическая оценка коэффициента эксцесса

In [25]:
from scipy.stats import t

gamma_values = [0.95, 0.99]
k = 7

for gamma in gamma_values:
    t_gamma = t.ppf((1 + gamma) / 2, df=k-1)

    margin_of_error = t_gamma * s / np.sqrt(k)
    confidence_interval = (x_avg_b - margin_of_error, x_avg_b + margin_of_error)

    print(f"Точность и доверительный интервал для математического ожидания для γ = {gamma}: {margin_of_error}, {confidence_interval}")

Точность и доверительный интервал для математического ожидания для γ = 0.95: 53.5807482315862, (391.7212319664336, 498.882728429606)
Точность и доверительный интервал для математического ожидания для γ = 0.99: 81.18264141979041, (364.1193387782294, 526.4846216178103)


Выводы: для уровня доверия 0.95 интервал уже (107.161), но всё ещё достаточно широкий, что указывает на значительную неопределённость, для уровня доверия 0.99 интервал значительно шире (162.365), что обеспечивает большую уверенность, но снижает точность оценки.

In [26]:
q_values = {0.95: 0.92, 0.99: 1.62}

for gamma, q in q_values.items():
    if gamma == 0.95:
        s_trust_interval = [s * (1 - q), s * (1 + q)]
    elif gamma == 0.99:
        s_trust_interval = [0, s * (1 + q)]

    print(f"Доверительный интервал для среднеквадратичного отклонения для γ = {gamma}: ({s_trust_interval[0]}, {s_trust_interval[1]})")

Доверительный интервал для среднеквадратичного отклонения для γ = 0.95: (4.634783547767192, 111.23480514641267)
Доверительный интервал для среднеквадратичного отклонения для γ = 0.99: (0, 151.78916118937562)


Выводы: для уровня доверия 0.95 интервал уже (106.6), но всё ещё достаточно широкий, что указывает на значительную неопределённость, для уровня доверия 0.99 интервал значительно шире (151.789), что обеспечивает большую уверенность, но снижает точность оценки.

In [27]:
from scipy.stats import norm

intervals = list(bin_edges)
norm_intervals = [(boundary - x_avg_b) / s for boundary in intervals]
norm_intervals[0] = -np.inf
norm_intervals[-1] = np.inf

theor_prob = [
    norm.cdf(norm_intervals[i]) - norm.cdf(norm_intervals[i - 1])
    for i in range(1, len(norm_intervals))
]
print("Теоретические вероятности:", theor_prob, sum(theor_prob))

theor_freq = [prob * len(data['nu']) for prob in theor_prob]
print("\nТеоретические частоты:", theor_freq, sum(theor_freq))

squares_diff = [
    (observed - theor) ** 2
    for observed, theor in zip(absolute_freq, theor_freq)
]
print("\nКвадраты разностей между наблюдаемыми и теоретическими частотами", squares_diff)

relative_squares_diff = [
    squares_diff / theor
    for squares_diff, theor in zip(squares_diff, theor_freq)
]
print("\nОтносительные квадраты разностей", relative_squares_diff)

squares_observed_freq = [ni**2 for ni in absolute_freq]
print("\nКвадраты наблюдаемых частот", squares_observed_freq)

relative_squares_observed_freq = [
    ni**2 / ni_prime
    for ni, ni_prime in zip(absolute_freq, theor_freq)
]
print("\nОтносительные квадраты наблюдаемых частот:", relative_squares_observed_freq, sum(relative_squares_observed_freq))

observed_X2 = sum(relative_squares_diff)
print("\nНаблюдаемое значение критерия Пирсона:", observed_X2)

Теоретические вероятности: [0.06815947485617586, 0.1389564166206667, 0.23591125328829138, 0.2588724816646606, 0.18362043620960244, 0.08416692502519374, 0.030313012335409284] 1.0

Теоретические частоты: [6.884106960473762, 14.034598078687338, 23.82703658211743, 26.14612064813072, 18.545664057169848, 8.500859427544567, 3.0616142458763376] 101.0

Квадраты разностей между наблюдаемыми и теоретическими частотами [4.477003354715582, 4.139589341798208, 0.6839895081604797, 1.3135925400715787, 11.932436805928276, 2.2525790212494052, 0.0037963152949097813]

Относительные квадраты разностей [0.6503390171624347, 0.29495603070276066, 0.02870644470633938, 0.050240437491651066, 0.6434084414095237, 0.264982504468969, 0.0012399717893993361]

Квадраты наблюдаемых частот [81, 144, 529, 625, 484, 49, 9]

Относительные квадраты наблюдаемых частот: [11.766232056688672, 10.260357952015422, 22.20166986258891, 23.904119789360934, 26.097744384239675, 5.764123076924402, 2.9396257259130616] 102.93387284773108

На

Оформим вычисления в таблицу:

Таблица 1

|    $ i $   	| $ [x_{i}, x_{i+1}) $ 	| $ n_i $ 	|        $ p_i $       	|      $ n'_i $      	|   $ (n_i - n'_i)^2 $  	| $ \cfrac{(n_i - n'_i)^2}{n'_i} $ 	| $ n_i^2 $ 	| $ \cfrac{n_i^2}{n'_i} $ 	|
|:----------:	|:--------------------:	|:-------:	|:--------------------:	|:------------------:	|:---------------------:	|:--------------------------------:	|:---------:	|:-----------------------:	|
|      1     	|    [320.0, 359.0)    	|    9    	|  0.06815947485617586 	|  6.884106960473762 	|   4.477003354715582   	|        0.6503390171624347        	|     81    	|    11.766232056688672   	|
|      2     	|    [359.0, 398.0)    	|    12   	|  0.1389564166206667  	| 14.034598078687338 	|   4.139589341798208   	|        0.29495603070276066       	|    144    	|    10.260357952015422   	|
|      3     	|    [398.0, 437.0)    	|    23   	|  0.23591125328829138 	|  23.82703658211743 	|   0.6839895081604797  	|        0.02870644470633938       	|    529    	|    22.20166986258891    	|
|      4     	|    [437.0, 476.0)    	|    25   	|  0.2588724816646606  	|  26.14612064813072 	|   1.3135925400715787  	|       0.050240437491651066       	|    625    	|    23.904119789360934   	|
|      5     	|    [476.0, 515.0)    	|    22   	|  0.18362043620960244 	| 18.545664057169848 	|   11.932436805928276  	|        0.6434084414095237        	|    484    	|    26.097744384239675   	|
|      6     	|    [515.0, 554.0)    	|    7    	|  0.08416692502519374 	|  8.500859427544567 	|   2.2525790212494052  	|         0.264982504468969        	|     49    	|    5.764123076924402    	|
|      7     	|    [554.0, 593.0]    	|    3    	| 0.030313012335409284 	| 3.0616142458763376 	| 0.0037963152949097813 	|       0.0012399717893993361      	|     9     	|    2.9396257259130616   	|
| $ \Sigma $ 	|           -          	|   101   	|          1.0         	|        101.0       	|           -           	|         1.933872847731078        	|     -     	|    102.93387284773108   	|

In [28]:
print(sum(relative_squares_observed_freq) - sum(absolute_freq), "=", observed_X2)

1.933872847731081 = 1.933872847731078


$ df = k - 3 = 4 $

Из таблицы в Приложении 5:

$ \chi^2_{крит} = 9.5 $

$ \chi^2_{набл} < \chi^2_{крит} (1.9 < 9.5) $

Выводы: гипотеза о нормальности распределения не отвергается.

## Выводы

В ходе выполнения практической работы были получены навыки вычисления интервальных статистических оценок параметров распределения выборочных данных и проверки «справедливости» статистических гипотез.

Результаты показали, что данные имеют значительный разброс, что подтверждается широкими доверительными интервалами для математического ожидания и среднеквадратичного отклонения. С увеличением уровня доверия с 0.95 до 0.99 интервалы становятся шире, что дает менее точную оценку параметров распределения.

Результаты проверки гипотезы о нормальности с использованием критерия Пирсона на уровне значимости 0.05 показали, что она не отвергается, то есть данные согласуются с предположением о нормальном распределении.